### ISSUE TITLE
WebSocketException when disconnecting only open market data stream

### ISSUE THREAD

**Issue by icgurney** [Petersoj/alpaca-java#87]
I believe the issue is `.getContainer()` on the userSession returns null after the userSession is closed in AbstractWebsocketClientEndpoint.

I'm using your example code in the README to reproduce this issue.

```net.jacobpeterson.abstracts.websocket.exception.WebsocketException: null
	at net.jacobpeterson.alpaca.websocket.marketdata.client.MarketDataWebsocketClient.removeListener(MarketDataWebsocketClient.java:150) ~[alpaca-java-6.1.jar:na]
	at net.jacobpeterson.alpaca.AlpacaAPI.removeMarketDataStreamListener(AlpacaAPI.java:1735) ~[alpaca-java-6.1.jar:na]
```

**Comment by Petersoj** [comment]
Interesting. I'll fix that this weekend. Thanks for reporting it.
Edit: Got busy last weekend. I'll fix this along with implementing the new Data V2 API.

**Comment by Petersoj** [comment]
Should be fixed in [7.0](https://github.com/Petersoj/alpaca-java/releases/tag/7.0)
